src/driver.rs

```diff
@@ -83,11 +83,6 @@ fn track_files(parse_sess: &mut ParseSess, conf_path_string: Option<String>) {
         file_depinfo.insert(Symbol::intern("Cargo.toml"));
     }
 
-    // `clippy.toml`
-    if let Some(path) = conf_path_string {
-        file_depinfo.insert(Symbol::intern(&path));
-    }
```

Comment:
It'd be worth leaving a comment behind mentioning that `clippy.toml` is being loaded elsewhere